client/my-sites/checkout/checkout/checkout-container.jsx

```diff
@@ -56,7 +56,6 @@ class CheckoutContainer extends React.Component {
 	shouldDisplaySiteCreatedNotice() {
 		return (
 			this.props.isComingFromSignup &&
-			this.props.selectedSite &&
```

Comment:
Good call, I didn't notice how `''` was being passed as the default parameter to `isSiteCreatedDateNew`